package.json

```diff
@@ -33,7 +34,7 @@
     "pretest": "npm run lint && npm run depcheck",
     "test": "nyc mocha \"src/**/*.test.ts\"",
     "test:ci": "npm test -- --reporter=xunit --reporter-options output=./reports/generator.xml",
-    "updateApis": "ts-node src/updateApis.ts"
+    "updateApis": "npm run backupApis && ts-node src/updateApis.ts && npm run diffApis"
```

Comment:
Is there a command that updates the APIs without diffs? if not would it make sense to add one

Reply:
do we have a use case for just download? We almost always want to diff. 
We can add a command to do backup and then update APIs (without diffs) if it is useful. This way we will able to run diffs if required after the download.